api-catalog-ui/frontend/src/components/Wizard/configs/wizard_regex_restrictions.jsx

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line import/prefer-default-export
+export const wizRegex = {
+    gatewayUrl: { value: '^(/[a-z]+\\/v\\d+)$', tooltip: 'Format: /api/vX, Example: /api/v1' },
+    version: { value: '^(\\d+)\\.(\\d+)\\.(\\d+)', tooltip: 'Semantic versioning expected, example: 1.0.7' },
```

Comment:
I've tried out this regex but it doesn't work

Reply:
^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(-[a-zA-Z\d][-a-zA-Z.\d]*)?(\+[a-zA-Z\d][-a-zA-Z.\d]*)?$
this works